### ISSUE TITLE
Feature: Bundled SSR patterns

### ISSUE THREAD

**Issue by drewnoakes** [xunit/resharper-xunit#68]
It'd be nice to have some SSR (structural search/replace) patterns for some CA/QF actions on Xunit code. Are there any already?

For example:

```csharp
// before
Assert.AreEqual(null, Foo());
// after
Assert.Null(Foo());
```

If there's interest in this feature, I'll endeavour to capture some patterns.

**Comment by citizenmatt** [comment]
Yep. There's always interest! Figuring out what the patterns should be is the tricky/time consuming thing. If you (or anyone else) could suggest some, that'd be great!

(Another one might be a pattern or set of patterns to convert from nunit to xunit)
